src/main/java/sk/freemap/gpxAnimator/ui/DurationFormatter.java

```diff
@@ -0,0 +1,89 @@
+package sk.freemap.gpxAnimator.ui;
```

Comment:
Please insert the Apache license file header. Quick solution: Copy from another file... :-)